app/components/ui/files/destinate-folder-modal/component.js

```diff
@@ -1,9 +1,10 @@
-import Ember from 'ember';
+import Component from '@ember/component';
+import Object from '@ember/object';
```

Comment:
There does not appear to be an `Object` exported from `@ember/object`, so I don't actually know which code this is importing... Is this using [`EmberObject`](https://www.emberjs.com/api/ember/3.2/classes/EmberObject) or the browser's built-in [`Object` class](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)

See https://www.emberjs.com/api/ember/3.2/modules/@ember%2Fobject